NMSReportingSuite/src/main/java/com/beehyv/nmsreporting/controller/UserController.java

```diff
@@ -510,12 +513,13 @@ public String forgotPassword(@RequestBody ForgotPasswordDto forgotPasswordDto, H
 
 
 
-
-    @RequestMapping(value = {"/deleteUser/{id}"}, method = RequestMethod.GET)
+//changed delete user to post, added a token verification
+    @RequestMapping(value = {"/deleteUser"}, method = RequestMethod.POST)
     @ResponseBody
-    public Map deleteExistingUser(@PathVariable("id") Integer id) {
+    public Map deleteExistingUser(HttpServletRequest request, @RequestBody Integer id) {
         User currentUser = userService.getCurrentUser();
-        if(currentUser != null){
+        String token = "dhty" + currentUser.getUserId().toString() + "alkihkf";
```

Comment:
is 'dhty' and 'alkijkf' random by choice? or can it be more meaningful?

Reply:
It's random by choice. They just wanted some token verification, so I am using a random string combined with userId